src/Runner.Worker/Container/DockerCommandManager.cs

```diff
@@ -87,9 +87,9 @@ public async Task<int> DockerPull(IExecutionContext context, string image)
             return await ExecuteDockerCommandAsync(context, "pull", image, context.CancellationToken);
         }
 
-        public async Task<int> DockerBuild(IExecutionContext context, string workingDirectory, string dockerFile, string tag)
+        public async Task<int> DockerBuild(IExecutionContext context, string workingDirectory, string dockerFile, string dockerContext, string tag)
         {
-            return await ExecuteDockerCommandAsync(context, "build", $"-t {tag} \"{dockerFile}\"", workingDirectory, context.CancellationToken);
+            return await ExecuteDockerCommandAsync(context, "build", $"-t {tag} -f \"{dockerFile}\" {dockerContext}", workingDirectory, context.CancellationToken);
```

Comment:
do we need to quote `{dockerContext}`?

Reply:
Probably. It works either way with my samples but I think if theres a whitespace or maybe other weird characters in the path it could get split / not work without quotes so I added them